calico-cloud/threat/web-application-firewall.mdx

```diff
@@ -65,123 +64,111 @@ kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"policySync
+  level=warning msg="WAF Process Http Request [2f435aca-4a3e-4f96-a7b2-a60c9745bd7b] URL '/test/artists.php?artist=0+div+1+union%23foo*%2F*bar%0D%0Aselect%23foo%0D%0A1%2C2%2Ccurrent_user' OWASP Warning'[2] Host:'10.224.0.38' File:'/etc/modsecurity-ruleset/REQUEST-942-APPLICATION-ATTACK-SQLI.conf' Line:'45' ID:'942100' Data:'' Severity:'0' Version:'OWASP_CRS/3.3.2' Message:'Warning. detected SQLi using libinjection.''"
 ```
 
-To enable seeing WAF logs in Kibana and Manager UI, see the [Seeing WAF events in Kibana and Manager UI](#seeing-waf-events-in-kibana-and-manager-ui) section.
-
-### Add and edit rules
-
-The OWASP Core Rule set that is bundled with {{prodname}} WAF, has reasonable defaults to get started.
-
-ModSecurity provides the following rule sets and options:
-
-- **DetectionOnly**
-  Allows all traffic to pass, regardless of action specified. However, potentially malicious traffic warning(s) are returned from ModSecurity and logged accordingly with the OWASP violation details.
-- **On or Off**
-  Denies all traffic if the SecAction is set to "deny" or "drop". However "block" traffic is not denied or dropped - this is slightly counter-intuitive.
-
-In case of an error, HTTP request will return HTTP 403 Response Code from Envoy to the originating service.
-
-| Action | Description                                                                                                                                                             | Disruptive? |
-| ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
-| Block  | Despite the name, this **will not block or drop the request**. ModSecurity will return detection=0 in this case and Calico will **not** log the event in Elasticsearch. | No          |
-| Deny   | Denies HTTP traffic as ModSecurity will return detection=1 and will log the event in Elasticsearch.                                                                     | Yes         |
-| Drop   | Denies HTTP traffic as ModSecurity will return detection=1 and will log the event in Elasticsearch.                                                                     | Yes         |
+#### Kibana
 
-#### Add or edit a rule set
+To view WAF events In Kibana:
 
-Create a directory, and download the core rules set files that you want to use, for example:
+1. Create the `tigera_secure_ee_waf*` index pattern. For help with index patterns, see [Kibana index pattern](https://docs.tigera.io/calico-cloud/visibility/kibana).
```

Comment:
Should this URL should have `docs.tigera.io` (and `calico-cloud`) here? I would have thought those should be {{something}} or a relative link.

Reply:
Can I put something like this? [Kibana index pattern] (../visibility/kibana.mdx)?

Reply:
yes, I think that should work - try it and let us know!